src/styles/blocks/card.css

```diff
@@ -81,12 +80,13 @@
   background-color: #00acdc;
   border: 1px solid transparent;
   border-radius: 5px;
-  padding: 12px 55px;
+  padding: 12px 56px 12px 55px;
+  transition-duration: 300ms;
+  transition-timing-function: ease-in;
```

Comment:
now you add transition-duration for `all` properties, it's hard for the browser. So always add transition property